azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -37,6 +42,8 @@
 
   private void setupAllMetrics() {
     dbConnectionMeter = MetricsUtility.addMeter("DB-Connection-meter", registry);
+    flowFailMeter = MetricsUtility.addMeter("flow-fail-meter", registry);
```

Comment:
The issue is more general than what's described in the issue. All metrics in the common class will subject to this issue.